modules/deeptools_bamFingerprint.cfmod.pl

```diff
@@ -0,0 +1,133 @@
+#!/usr/bin/env perl
```

Comment:
I think it makes more sense to have keep the two modules separate because the do different things and give different output.

bamCoverage is the main thing here. It creates a bigwig coverage track, which is something people use quite often. bamFingerprint just creates a diagnostic plot of how unevenly read coverage is distributed over the genome. I would guess that there are many situations where you just want the coverage track, and are not very interested in the diagnostic plot for example.